Examplereorg

DanielRPhillips opened the pull request:
This is pull request to bring organization of BAND Materials into three "buckets": software, BANDsoftware_uses, and examples, into develop. 

Requesting review from @furnstahl and @wildsm

DanielRPhillips committed:
Rewrote README.md for examples directory

DanielRPhillips committed:
Made new README for the main software directory

DanielRPhillips committed:
Added jitr to list of BANDframework tools

DanielRPhillips committed:
First pass at README file for BANDtool_examples

DanielRPhillips committed:
Fixed broken link in BANDtool_examples README

DanielRPhillips committed:
Fix broken link in BANDtool_examples

DanielRPhillips committed:
Maybe I finally got the link in BANDtool_examples right this time?

Commit:
Reorganized BAND software into three new categories

Commit:
Submoduled SAMBA, QGP_Bayes, nuclear-saturation into examplereorg branch

Commit:
Renamed BANDtool_examples to BANDsoftware_uses

DanielRPhillips committed:
Updated README for software for new name for applications thereof

Commit:
Moved BMEX to examples and updated README accordingly

DanielRPhillips committed:
Update examples README to include BMEX

DanielRPhillips committed:
Update main README

DanielRPhillips committed:
Updating main README

DanielRPhillips committed:
Updating main README to give correct location of submodules

wildsm committed:
Update README.md

Fixed typo found by the spell action.

wildsm committed:
Update .wordlist.txt

Added BANDsoftware

wildsm commented:
I have fixed the typo and added BANDsoftware to the OK list; the CI actions now pass.

wildsm committed:
Update README.md

Correcting some minor issues. 
Making sure version is not increased until we are staging for v0.5.0.

wildsm commented:
- [x] ToDo: Decide whether jitr is in examples or software. Then update the listing (and alphabetize submodules list) in README.

wildsm committed:
Update README.md

Fixies:
- the link to BFRESCOX
- the version number
- the link to software (so that it tracks the correct branch)
- nuclear physics

wildsm committed:
Update README.md

Fixed some links and version.

wildsm committed:
Update release-proc.rst

Adding three READMEs to the list of places that now contain a version number!

wildsm committed:
Update README.md

Fixed version number and some links.

wildsm committed:
Update README.md

Moved jitr to software consistently

wildsm reviewed (approved):
I have complete a review of the reorg and pushed several changes throughout.

The one thing that is still a bit odd is that in several places (whenever there is a submodule), the destination repo is being linked and some times this link may go to a version of the repo that is not the precise destination of the submodule. We might want to consistently be linking the submodule link and then as part of the dev guide telling people that when they update the submodule they need to update all the relevant links....

wildsm committed:
Merge branch 'examplereorg' into swpatch1

wildsm committed:
Bringining in latest develop

Bringing in latest develop

wildsm commented:
This branch has been updated to reflect lcgp now in develop!
